templates/zerver/dev_login.html

```diff
@@ -14,23 +17,35 @@ <h4 class="login-page-header">{{ _('Click on a user to log in!') }}</h4>
                 <div class="controls">
                     <div class="group">
                         <h2>{{ _('Administrators') }}</h2>
-                        {% for user_email in direct_admins %}
-                        <p><input type="submit" name="direct_email" class="btn-direct btn-admin" value="{{ user_email }}" /></p>
-                        {% endfor %}
+                        {% if direct_admins %}
+                            {% for user_email in direct_admins %}
+                            <p><input type="submit" name="direct_email" class="btn-direct btn-admin" value="{{ user_email }}" /></p>
+                            {% endfor %}
+                        {% else %}
+                            <p>{{ _('No administrators found in this realm') }}</p>
+                        {% endif %}
                     </div>
 
                     <div class="group">
                         <h2>{{ _('Normal users') }}</h2>
-                        {% for user_email in direct_users %}
-                        <p><input type="submit" name="direct_email" class="btn-direct btn-user" value="{{ user_email }}" /></p>
-                        {% endfor %}
+                        {% if direct_users %}
+                            {% for user_email in direct_users %}
+                            <p><input type="submit" name="direct_email" class="btn-direct btn-admin" value="{{ user_email }}" /></p>
+                            {% endfor %}
+                        {% else %}
+                            <p>{{ _('No normal users found in this realm ') }}</p>
+                        {% endif %}
                     </div>
 
                     <div class="group">
                         <h2>{{ _('Community users') }}</h2>
-                        {% for user_email in community_users %}
-                        <p><input type="submit" name="direct_email" class="btn-direct btn-user" value="{{ user_email }}" /></p>
-                        {% endfor %}
+                        {% if community_users %}
+                            {% for user_email in community_users %}
+                            <p><input type="submit" name="direct_email" class="btn-direct btn-admin" value="{{ user_email }}" /></p>
+                            {% endfor %}
+                        {% else %}
+                            <p>{{ _('No community users found in this realm ') }}</p>
```

Comment:
No need to tag for translations here; we don't tag development-only code for translation.